platform/src/main/java/org/stellar/anchor/platform/PlatformServer.java

```diff
@@ -25,6 +26,7 @@
       "org.stellar.anchor.platform.component.platform",
       "org.stellar.anchor.platform.component.share"
     })
+@EnableScheduling
```

Comment:
❓ Is scheduling used in the `PlatformServer`?

Reply:
Yes. Platform server runs TrustlineCheckJob.